RxJavaTemplate/app/src/main/java/co/nimblehq/rxjava/extension/SharePreferencesExt.kt

```diff
@@ -0,0 +1,10 @@
+package co.nimblehq.rxjava.extension
+
+import android.content.SharedPreferences
+
+fun SharedPreferences.execute(operation: (SharedPreferences.Editor) -> Unit) {
```

Comment:
Do we still need this? 🤔  Why don't we call `sharedPreferences.clear()` from the Base class? 🤔

Reply:
yeah we need it @hoangnguyen92dn, this is an extension function that helps us don't have to rewrite all the repeated calls like apply(), edit(), etc. and just need to write the main operation in the operation block

Reply:
Ah. I missed it. So it's oke 😄